src/views/AuthorityView/AuthorityView.test.js

```diff
@@ -78,4 +88,32 @@ describe('Given AuthorityView', () => {
       expect(mockHistoryPush).toHaveBeenCalled();
     });
   });
+
+  describe('when user cliked edit shortcuts', () => {
```

Comment:
It would be nice to remove the typo as well :)
```suggestion
  describe('when user clicked edit shortcuts', () => {
```